### DISCUSSION TITLE
Proscan integration

### DISCUSSION THREAD

**Discussion by ryanhowell18** [chuot/rdio-scanner#429] [Q&A]
Hello everyone, I believe I have asked questions in relation to Proscan previously however I was wondering if anyone has managed to get this working with Proscan and if so would they be willing to share their settings? 

I love Rdio-scanner but struggle without the Proscan element of it. 

Thanks in advance.

**Comment by noamlivne** [comment]
I did not succeed directly with my Proscan setup alone (the calls were cut off or doubled).
So I followed this guide to intermediate with Trunking Recorder:
https://www.scannerbox.us/TrunkingRecorder/ProScan/
So if I remember correctly, my setup was:
Scanner -> ProScan -> Trunking Recorder -> Rdio Scanner

**Reply by ryanhowell18** [reply]
Hello, thankyou for the information would it be possible for you to share some photos of the setup on the call uploading on trunking recorder as I can not seem to get it to work?

**Reply by noamlivne** [reply]
If I remember correctly I only followed the instructions I mentioned.
I did it about a year ago I think, perhaps with older versions, so maybe some things have changed.
Attached are the pics of the things I changed.
<img width="576" alt="Call uploading" src="https://github.com/user-attachments/assets/d80b9369-5da1-46fc-b4a5-c40f51a1d40e">
<img width="576" alt="Call import" src="https://github.com/user-attachments/assets/c59c93aa-77a4-4bc7-b129-61905984ffda">

**Reply by ryanhowell18** [reply]
Hmm thanks for that! I have the same but for some reason I can not seem to get the TG's to show under the section that you have cropped if that makes sense, Iknow I am being a pain could you scroll across on the top section in the Trunking Recorder so I can see the rest to make sure mine are correct? 

Another thing, I'm guessing the scanner that was connected to pro scan was scanning and not holding on a frequency? 

Sorry for all the questions! 

Thanks for all the help.

**Reply by noamlivne** [reply]
My setup is probably different than anybody's else + I am in another country.
I have an old Analog Uniden bcd396xt that its screen does not work any more.
I Scan a few memory banks.
I connect it to an old version of ProsScan 16.0 with a USB cable + an Audio (male to male) cable.
Here are pics of the Recorder tab of ProScan, the Audio Control tab of ProScan, the Options -> Recorder tab of ProScan, and the Options -> Recording Text tab from ProScan.

I do not think I changed anything else in Trunking Recorder - and I followed their instructions.
Just in case here are the rest of the screen pics.
<img width="450" alt="Trunking Recorder Metadata" src="https://github.com/user-attachments/assets/ab97fcb6-5714-448a-90e1-bbee89bb22a7">
<img width="448" alt="Trunking Recorder Streaming" src="https://github.com/user-attachments/assets/10f741e0-ac82-48eb-adad-8a5d4490b7a7">
<img width="452" alt="Trunking Recorder Favoritespng" src="https://github.com/user-attachments/assets/6d1f0c6b-770c-4c05-ba56-f36e5a097c00">
![Trunking Recorder Webserver](https://github.com/user-attachments/assets/d12bbf85-1d7a-493a-9cc2-88859a72e7e9)
![Trunking Recorder Recording](https://github.com/user-attachments/assets/fc2590e1-cfbc-4cc4-97d6-05f15fab6f13)
<img width="447" alt="Trunking Recorder General" src="https://github.com/user-attachments/assets/e5d78e8a-f539-4794-a5e1-893301a597e4">
![Options Recording Text](https://github.com/user-attachments/assets/9ed77324-2678-4d68-abc2-060cb86dfe46)
![Options Recorder](https://github.com/user-attachments/assets/e8c70c1c-492b-4e67-92b6-70e7358cc771)
![Audio Control](https://github.com/user-attachments/assets/970d8fab-78cc-45c2-8743-5b7bd1d71bd0)
![Recorder](https://github.com/user-attachments/assets/704f0431-293e-44d3-8e67-f27c55166610)
